h2o-core/src/main/java/hex/Model.java

```diff
@@ -704,6 +707,7 @@ public void setNames(String[] names, String[] columntypes) {
     public long _end_time;
     public long _run_time;
     public long _total_run_time; // includes building of cv models
+    public static int INFORMATION_TABLE_NUM_ROWS = 2;
```

Comment:
is this constant necessary? is it used somewhere else other than the getInformationTableNumRows method?